changelog/v1.2.0-beta2/peer-auth-api.yaml

```
@@ -0,0 +1,5 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo-mesh-enterprise/issues/833
+    resolvesIssue: false
+    description: Add api changes for peer auth creation.
```

Would it be important to mention the matching TLS settings?
```suggestion
    description: Add API changes for creating peer authentication, including matching mTLS settings for Istio mesh connections.
```